components/script/dom/window.rs

```diff
@@ -319,10 +319,15 @@ impl Window {
         &self.image_cache_thread
     }
 
+    /// Retrieve unsafely the browsing context
     pub fn browsing_context(&self) -> Root<BrowsingContext> {
         self.browsing_context.get().unwrap()
     }
 
+    pub fn safe_browsing_context(&self) -> Option<Root<BrowsingContext>> {
```

Comment:
Please rename method to something like `maybe_browsing_context` or something like that. Safety is a different concept.